ci/docker/runtime_functions.sh

```diff
@@ -427,6 +427,12 @@ unittest_ubuntu_cpu_scala() {
     make scalatest USE_BLAS=openblas
 }
 
+unittest_ubuntu_gpu_scala() {
+    set -ex
+    make scalapkg USE_OPENCV=1 USE_BLAS=openblas USE_CUDA=1 USE_CUDA_PATH=/usr/local/cuda USE_CUDNN=1
```

Comment:
set a environment variable called SCALA_TEST_ON_GPU=1, in the test you can check for this variable and change the context to gpu(0)

Reply:
Added to the test section